docs/known_issues.rst

```diff
@@ -139,6 +139,8 @@ A workaround for this at the moment would be to do::
 
 As well as with `~numpy.full` one cannot do `~numpy.zeros`, `~numpy.ones`, and `~numpy.empty`.
 
+See: https://github.com/astropy/astropy/issues/11582
```

Comment:
Instead of giving a link to an issue that will be closed, how about explicitly mentioning `np.arange` (perhaps even including the error, like `np.full`)? We can then also mention that `np.linspace` works fine.

Reply:
Why would the issue be closed? It would only be closed if and when this "known issue" is fixed.

Reply:
There is a chance we can close the issue as "won't fix" 😆

Reply:
OK, but right now this PR has been set to automatically close the issue. Can we change "Closes" to "Refs" so that it doesn't do that?

Reply:
Done. Sorry. That was my bad.

Reply:
Please take another look. I added the exception and workarounds as suggested by @mhvk.